src/Html/DataContainer.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\RadBundle\Html;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class DataContainer
+{
+    /**
+     * Renders the data container to HTML.
+     *
+     * @param array       $data
+     * @param string      $className
+     * @param string|null $id
+     *
+     * @return string
+     */
+    public function renderToHtml (array $data, string $className, ?string $id = null)
+    {
+        return \sprintf(
+            '<script%s class="_data-container %s" type="application/json">%s</script>',
+            null !== $id ? ' id="' . \htmlspecialchars($id, \ENT_QUOTES) . '"' : "",
+            \htmlspecialchars($className, \ENT_QUOTES),
+            \htmlspecialchars(\json_encode($data), \ENT_NOQUOTES)
+        );
+    }
+
+
+    /**
+     * Renders the data container as a direct response.
+     * Convenient to use in embedded controllers.
+     *
+     * @param array       $data
+     * @param string      $className
+     * @param string|null $id
+     *
+     * @return Response
+     */
+    public function createResponse (array $data, string $className, ?string $id = null) : Response
```

Comment:
Shouldn't this be a `static` function instead?

Reply:
I would like to avoid static functions if I don't really need them.

The above one could also be static, but then it would be impossible to change it in the future, for example if the class needs a service etc.